bcda/models/alr.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+import (
+	"time"
+)
+
+type Alr struct {
```

Comment:
Should we add a unique on MetaKey + BeneMBI?

Reply:
I don't think so?

If you look at table 1-2 (as well as 1-3 and etc.) in the data dictionary:

"Each record represents a unique MBI-participant TIN combination."  Which sounds like there can be more than one row per MBI.

This just reminded me though. Are we in the business of remove duplicate rows?

Reply:
Ah good find! I think we should discuss further about cleanup in general.